lib/modules/@apostrophecms/storybook/index.js

```diff
@@ -3,18 +3,48 @@ const fs = require('fs-extra');
 const IMPORTS_PLACEHOLDER = /(\/\/ IMPORTS)/g;
 const COMPONENTS_PLACEHOLDER = /(\/\/ COMPONENTS)/g;
 const STORIES_PLACEHOLDER = /"STORIES"/g;
+const childProcess = require('child_process');
 
 module.exports = {
   init(self, options) {
-    self.addTask('build', 'update the data/temp/.storybook folder', self.build);
-    self.addTask('run', 'start storybook', self.run);
+    self.addTask('build', 'pass a folder name to build a static storybook', self.build);
+    self.addTask('run', 'start storybook dev server', self.run);
+    self.addTask('deploy', 'deploy static storybook to github pages of your project repo', self.deploy);
   },
   methods(self, options) {
     return {
       async build(apos, argv) {
+        const target = require('path').resolve(process.cwd(), argv._[1]);
+        if (!target) {
+          throw 'Must specify target directory as first argument.';
+        }
+        await self.preBuild(argv);
+        const buildDir = self.getBuildDir();
+        process.env.APOS_ROOT = self.apos.rootDir;
+        childProcess.execSync(`cd ${buildDir} && npx build-storybook -s ./public -c .storybook -o ${target}`, { stdio: 'inherit' });
+      },
+      async run(apos, argv) {
+        await self.preBuild(argv);
+        const buildDir = self.getBuildDir();
+        process.env.APOS_ROOT = self.apos.rootDir;
+        childProcess.execSync(`cd ${buildDir} && npx start-storybook -s ./public -p 9001 -c .storybook`, { stdio: 'inherit' });
+      },
+      async deploy(apos, argv) {
+        await self.preBuild(argv);
+        const buildDir = self.getBuildDir();
+        process.env.APOS_ROOT = self.apos.rootDir;
+        require('child_process').execSync(`
+          cd ${buildDir} &&
+          npx build-storybook -s ./public -c .storybook -o $APOS_ROOT/data/temp/storybook-static &&
+          npx storybook-to-ghpages -e $APOS_ROOT/data/temp/storybook-static &&
```

Comment:
Did you test this deploy? When I used the `-e` option like this the images weren't included.

Reply:
Yes, it's on ui.apos.dev now. The avatar images are working. I'm not sure if I should be checking for anything else.